src/stdlib/Math.ts

```diff
@@ -1,6 +1,112 @@
 import { Interpreter } from "../interpreter";
 
+/** Returns the absolute value of a float. */
+export const Abs = new Callable(
+    "Abs",
+    {
+        signature: {
+            args: [{name: "x", type: ValueKind.Float}],
+            returns: ValueKind.Float
+        },
+        impl: (interpreter: Interpreter, x: Float) => new Float(Math.abs(x.getValue()))
+    }
+);
+
+/*
+ * Returns the integer as a 32-bit float. 
+ * ** NOTE: the function name implies it makes a 64-bit float, but the docs say
+ *     it currently returns a 32-bit float, but may return a 64-bit float in the future.
+ */
+export const Cdbl = new Callable(
+    "Cdbl",
+    {
+        signature: {
+            args: [{name: "x", type: ValueKind.Int32}],
+            returns: ValueKind.Float
+        },
+        impl: (interpreter: Interpreter, x: Int32) => new Float(x.getValue())
+    }
+);
+
+/** Returns an integer from a float rounding up from midpoints */
+export const Cint = new Callable(
+    "Cint",
+    {
+        signature: {
+            args: [{name: "x", type: ValueKind.Float}],
+            returns: ValueKind.Int32
+        },
+        impl: (interpreter: Interpreter, x: Int32) => new Int32(Math.round(x.getValue()))
+    }
+);
+
+/** Returns the integer as a 32-bit float. */
+export const Csng = new Callable(
+    "Csng",
+    {
+        signature: {
+            args: [{name: "x", type: ValueKind.Int32}],
+            returns: ValueKind.Float
+        },
+        impl: (interpreter: Interpreter, x: Int32) => new Float(x.getValue())
+    }
+);
+
+/** Returns an integer from a float removing fractional parts. */
+export const Fix = new Callable(
+    "Fix",
+    {
+        signature: {
+            args: [{name: "x", type: ValueKind.Float}],
+            returns: ValueKind.Int32
+        },
+        impl: (interpreter: Interpreter, x: Int32) => new Int32(Math.trunc(x.getValue()))
+    }
+);
+
+
+/** Returns an integer from a float. */
+export const Int = new Callable(
+    "Int",
+    {
+        signature: {
+            args: [{name: "x", type: ValueKind.Float}],
+            returns: ValueKind.Int32
+        },
+        impl: (interpreter: Interpreter, x: Int32) => new Int32(Math.floor(x.getValue()))
+    }
+);
+
+/** Returns -1 if parameter is negative, 0 if zero, and 1 if positive. */
+export const Sgn = new Callable(
+    "Sgn",
+    {
+        signature: {
+            args: [{name: "x", type: ValueKind.Float}],
+            returns: ValueKind.Int32
+        },
+        impl: (interpreter: Interpreter, x: Float) => {
+            let val = x.getValue();
+            if (val > 0.0) return new Int32(1);
+            else if (val < 0.0) return new Int32(-1);
+            else return new Int32(0);
+        }
+    },
+    {
+        signature: {
+            args: [{name: "x", type: ValueKind.Int32}],
+            returns: ValueKind.Int32
+        },
+        impl: (interpreter: Interpreter, x: Int32) => {
```

Comment:
Given that these are the same implementation, do you think it'd be worth pulling that out to a separate variable and using it in both signatures?  Something like this:

```typescript
function SgnImpl(interpreter: Interpreter, x: Int32 | Float) {
    let val = x.getValue();
    if (val > 0.0) return new Int32(1);
    else if (val < 0.0) return new Int32(-1);
    else return new Int32(0);
}

export const Sgn = new Callable(
    "Sgn",
    {
        signature: {
            // Accepts Float
        },
        impl: SgnImpl
    },
    {
        signature: {
            // Accepts Int32
        },
        impl: SgnImpl
    }
);
```

Reply:
I actually did that locally and thought it wasn't significant enough to justify, but if you'd prefer I can add something like this.